### DISCUSSION TITLE
Primary Key vs. Display Value [Mar 19]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#5369] [Discord Threads]
> 🙋‍♂️ **kyleecodes**  
#### I ran some tests, enabled pg logging, checked if primary key change is changing in db with pgadmin, but I don't see any primary key being set by doing so"  
![image0](https://cdn.discordapp.com/attachments/661905455894888493/1076213607227990127/image.png)

**Comment by o1lab** [comment]
> 🙋‍♂️ **kyleecodes**  
#### Your Display Value and Primary Keys are independent of each other, so they are not the same.    
####    
#### Display Value Columns can contain your Primary Keys IF you set them as your Display Values, but Display Values are for organizational purposes within the application itself.  
#  
> 🙋‍♂️ **kyleecodes**  
#### Learn more about the Display Value here: https://docs.nocodb.com/setup-and-usages/display-value  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/1076213607269937197)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
